scallop.yml

```diff
@@ -2,44 +2,67 @@
 #path/software [input_command <input.bam>] [other_options] [output_command <output.gtf>]
 command_format: ''
 
-#initial option of assemble software
-initial_option:
-  name: scallop
-  input_option: ' -i '
-  output_option: ' -o '
-  #software specific option
-  additional_option: ' --verbose 0 --min_transcript_coverage 0 --library_type empty'
+#full commands for black-box software
+#formats the specified value(s) and insert them inside the string's placeholder.
+#The placeholder is defined using curly brackets: {}
+#The placeholder should appear excatly once in the same section
+testing_software:
+  format: '{path} {input_option} {input_file} {parameters} {output_option} {pid}{output_format} {additional_option}'
+  input_option: '-i'
+  output_option: '-o'
+  output_format: '.gtf'
+  pid: ''
+  additional_option: ' --verbose 0 --min_transcript_coverage 0 --library_type empty > /dev/null 2>&1'
```

Comment:
Sending to `/dev/null` can create silent errors. Is there a better way to save those logs? Or are they really useless?

Reply:
Not sure if there is any silent error I missed, I will run default parameters without /dev/null on scallop and scallop2 to see if there is any. 
But these logs are indeed useless (if no error within them) BO computes AUC based following results produced by gffcompare and gtfcuff, which only depends on GTF files not log files.